test/cov.jl

```diff
@@ -88,62 +88,62 @@ end
 # weighted covariance
 
 if VERSION < v"0.5.0-dev+679"
-    @test cov(X, wv1)           ≈ S1w ./ sum(wv1)
-    @test cov(X, wv2; vardim=2) ≈ S2w ./ sum(wv2)
+    @test cov(X, wv1; corrected=false)           ≈ S1w ./ sum(wv1)
```

Comment:
As I just noted in a comment above, I think the best deprecation path is to deprecate `WeightsVec` but keep it around so that existing code does not use the correction, and point to new weights types in the deprecation warning.